interactions/context.py

```diff
@@ -27,8 +27,8 @@ class Context(DictSerializerMixin):
     :ivar Optional[Message] message?: The message data model.
     :ivar Member author: The member data model.
     :ivar User user: The user data model.
-    :ivar Channel channel: The channel data model.
-    :ivar Guild guild: The guild data model.
+    :ivar Union[Channel, MISSING] channel: The channel data model.
+    :ivar Union[Guild, None, MISSING] guild: The guild data model.
```

Comment:
Docstring on a sentinel value and a None return should always respect `Optional[]` typing schema.